### DISCUSSION TITLE
Service widget integration for Kuma

### DISCUSSION THREAD

**Discussion by kai-arsenault** [gethomepage/homepage#460] [Feature Requests] [answered]
Would love to see Kuma have service widget integration. I think being able to connect it to the fields below and show how many pages are up, down, unknown, or pause. Maybe there a more helpful way to display the info in a widget, but I think something like this could be cool and useful.

![Screenshot 2022-10-27 135038](https://user-images.githubusercontent.com/35471761/198362691-80223b40-2929-4c10-a18d-1b83e3a321e8.jpg)

**Comment by lonelyb3ar** [comment]
Oh this would be great little thing for us homepagers 

Love the work so far. 
Great stuff.

**Comment by divemasterjm** [comment]
+1

**Comment by shamoon** [comment, marked as answer]
See #123
